components/builder-admin/src/server/mod.rs

```diff
@@ -35,7 +35,7 @@ impl HttpGateway for AdminSrv {
 
     fn add_middleware(config: Arc<Self::Config>, chain: &mut iron::Chain) {
         chain.link(persistent::Read::<GitHubCli>::both(
-            GitHubClient::new(&*config),
+            GitHubClient::new((*config).github.clone()),
```

Comment:
Rust question time: What's the significance of the parentheses around `*config`?

Reply:
Actually now that I go to type this out it might be overkill. I was dereferencing the `Arc` to get at the inner value but `.github` will do the same thing

Reply:
Cool.